### DISCUSSION TITLE
I have found out a possible bug in the "profile section"

### DISCUSSION THREAD

**Discussion by anandpanda** [subrotokumar/kurumi#10] [General]
i am attaching the screenshot for the same below.

I saw from the preview images at the repository of yours', but I am not able to get the same result at my end.

Please look into it.

![Screenshot_2024-01-19-09-14-59-142_com.subrotokumar.kurumi.jpg](https://github.com/subrotokumar/kurumi/assets/148877270/0362fe62-24b8-4343-900b-20abb9a37dfc)

![Screenshot_2024-01-19-09-15-01-700_com.subrotokumar.kurumi.jpg](https://github.com/subrotokumar/kurumi/assets/148877270/cc10dc00-ec08-40ae-8a29-f5a78c7d1fc0)

![Screenshot_2024-01-19-09-14-55-727_com.subrotokumar.kurumi.jpg](https://github.com/subrotokumar/kurumi/assets/148877270/6721fe5b-ecd0-4f7c-9c33-22c4f3904b12)

**Comment by subrotokumar** [comment]
I checked your profile on Anilist. Currently, there are no anime, manga, or characters marked as favorites, which is why you can't find any entries on your profile screen. It's important to note that even after adding something as a favorite, it may take around 6 hours for the changes to reflect.
